### DISCUSSION TITLE
Example of WS_EX_NOACTIVATE

### DISCUSSION THREAD

**Discussion by wnunezc** [cefsharp/CefSharp#4194] [General]
Any example of implement this:

https://github.com/cefsharp/CefSharp/issues/1811

**Comment by amaitland** [comment]
WS_EX_NOACTIVATE is set by default currently. 

Review/debug the source if you want to know more.

**Reply by wnunezc** [reply]
I'm investigating, it happens that I've created a pseudo client in c# winform for a tester of applications developed in webgl. The problem is that for some reason, if I change the Windows to other program openend, in some cases the keyboard stops interacting with the browser component. and the webgl application stops receiving any keyboard interaction... it only receives mouse interactions... that's really weird...

**Reply by wnunezc** [reply]
I dont know how to debug if browser control is receiving the interaction like typewrite or event is any issues in the webgl app

**Reply by amaitland** [reply]
> The problem is that for some reason, if I change the Windows to other program openend, in some cases the keyboard stops interacting with the browser component

Sounds entirely unrelated to activation on browser creation (https://github.com/cefsharp/CefSharp/issues/1811)

CefSharp is built with sourcelink support, you can debug parts of the code directly from your application see https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/sourcelink
